### ISSUE TITLE
Using Otto with Wordpress

### ISSUE THREAD

**Issue by oskarrough** [hashicorp/otto#471]
Hi, I'm trying to use Otto with Wordpress. It auto-detects and installs almost everything out of the box only a MySQL extension is missing:

`Your PHP installation appears to be missing the MySQL extension which is required by WordPress.`

… which seems to be discussed here: https://groups.google.com/forum/#!topic/otto-tool/37IXF-ZX_J8.

Would it be fair to assume that Otto should take care of this automatically or would I need to do this manually?

**Comment by leowmjw** [comment]
@oskarrough Yes, it is a bug, Otto should have handled it for you. During the implicit detection of Wordpress as part of the PHP app type; the php5-mysql should have been added but this rule is missing in the code base.  For now, you will have to do by hand until a fix is out.

Additionally, the implicit MySQL Docker image should (for example) be passed the Environmental variables below so that a DB is automatically setup and ready for Wordpress setup script:
```
-e MYSQL_ROOT_PASSWORD=t0ps3cr3tpa55w0rdz -e MYSQL_USER=wordpress -e MYSQL_PASSWORD=passw0rd 
```

**Comment by oskarrough** [comment]
@leowmjw thank you. This is exactly the issue and steps I ran into. If I can help test the upcoming fix, let me know.

**Comment by coffeeneed** [comment]
@oskarrough Hi. May I ask how you added Wordpress as a dependency? I tried doing that according the docs, but I got this message:

**Fetching dependency: git::https://github.com/WordPress/WordPress.git
Error compiling Appfile: Dependency 'git::https://github.com/WordPress/WordPress.git' doesn't have an Otto ID yet!

An Otto ID is generated on the first compilation of the Appfile.
It is a globally unique ID that is used to track the application
across multiple deploys. It is required for the application to be
used as a dependency. To fix this, check out that application and
compile the Appfile with `otto compile` once. Make sure you commit
the .ottoid file into version control, and then try this command
again.**
